src/lib/arch/aarch64/Instruction_decode.cc

```diff
@@ -188,6 +188,8 @@ void Instruction::decode() {
     return;
   }
 
+  uint16_t sourceRegisterCount = 0;
```

Comment:
Can we not just use `sourceRegisters.size()`? Each increment of this variable seems to be paired with a push_back to sourceRegisters. May have missed something here

Reply:
Yup correct, variable removed